apps/web/src/styles/main.scss

```diff
@@ -33,3 +34,9 @@
 
 // Shame file used for everytihng you will be shamed for.
 @use "shame";
+
+// FIXME: govuk accessible autocomplete – copied from
+// node_modules/accessible-autocomplete for whatever reason none of
+// "~accessible-autocomplete", "~accessible-autocomplete/src/autocomplete" or
+// "~accessible-autocomplete/src/autocomplete.css" will work
+@use "autocomplete.css";
```

Comment:
In `apps/web/scripts/compile-css.js` line 39, we should update that with the entire node_modules folder not just govuk-frontend and then we can import it directly. I will raise a PR and you can inherit the changes.

Reply:
I merged a fix for this so you can follow that example.